Fix the build with kdelibs4 4.14.30.

commit f785317ba88528760886b592aa9604b0c4ba62dd
Author: David Faure <faure@kde.org>
Date:   Sun Jul 26 18:12:24 2015 +0200

    set cmake_min_req to 2.8.9 to match kdelibs4 policy; fix build accordingly

    (working around the lack of ${KDE4_KMEDIAPLAYER_LIBS})

    CCMAIL: kde-buildsystem@kde.org

--- CMakeLists.txt
+++ CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required( VERSION 2.6.2 FATAL_ERROR )
+cmake_minimum_required(VERSION 2.8.9)
 project(kphotoalbum)
 
 # provide drop-down menu for build-type in cmake-gui:
@@ -364,7 +364,7 @@ kde4_add_library(Utilities STATIC ${libUtilities_SRCS})
 target_link_libraries(kphotoalbum Utilities)
 
 # External components
-target_link_libraries(kphotoalbum ${KDE4_KIO_LIBS} ${JPEG_LIBRARY} kmediaplayer ${KDE4_PHONON_LIBS})
+target_link_libraries(kphotoalbum ${KDE4_KIO_LIBS} ${JPEG_LIBRARY} ${KDE4_TARGET_PREFIX}kmediaplayer ${KDE4_PHONON_LIBS})
 
 if(KIPI_FOUND)
   target_link_libraries(kphotoalbum ${KIPI_LIBRARIES})
